nix_update/update.py

```diff
@@ -82,24 +82,28 @@ def nix_prefetch(cmd: List[str]) -> str:
 
 
 def update_src_hash(opts: Options, filename: str, current_hash: str) -> None:
-    target_hash = nix_prefetch([f"(import {opts.import_path} {{}}).{opts.attribute}"])
+    target_hash = nix_prefetch(
+        [
+            f"(import {opts.import_path} (if (builtins.hasAttr "config" (builtins.functionArgs (import {opts.import_path}))) then { config.checkMeta = false; } else {})).{opts.attribute}"
+        ]
+    )
     replace_hash(filename, current_hash, target_hash)
 
 
 def update_mod256_hash(opts: Options, filename: str, current_hash: str) -> None:
-    expr = f"{{ sha256 }}: (import {opts.import_path} {{}}).{opts.attribute}.go-modules.overrideAttrs (_: {{ modSha256 = sha256; }})"
+    expr = f"{{ sha256 }}: (import {opts.import_path} {{ config.checkMeta = false; }}).{opts.attribute}.go-modules.overrideAttrs (_: {{ modSha256 = sha256; }})"
```

Comment:
I think we need the same check than for all the other variants as well.
Ideally factored out in a variable.